server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -1126,6 +1176,18 @@ def __init__(self, term: str | Field, date_format: str, alias: str | None = None
         super().__init__("PARSE_DATE", term, date_format, alias=alias)
 
 
+class RegexpFunction(functions.Function):
+    REGEXP_LIKE = "REGEXP_LIKE"
+    REGEXP_CONTAINS = "REGEXP_CONTAINS"
+    NOT_REGEXP_LIKE = "NOT REGEXP_LIKE"
+    NOT_REGEXP_CONTAINS = "NOT REGEXP_CONTAINS"
```

Comment:
I think this should live in the `RegexOp` enum, and it should be changed to `RegexOp(str, Enum)`

Reply:
Make sense, updated here : https://github.com/ToucanToco/weaverbird/pull/1418/commits/748984ee43328c4f6d25ac0abaf82c1f2bf513a7